src/sentry/integrations/gitlab/integration.py

```diff
@@ -92,6 +92,25 @@ def get_repositories(self, query=None):
         resp = self.get_client().search_group_projects(group, query)
         return [{"identifier": repo["id"], "name": repo["name_with_namespace"]} for repo in resp]
 
+    def get_stacktrace_link(self, repo, filepath, version):
+        project_id = repo.config["project_id"]
+        try:
+            # repos are projects in GL so the project_id is the repo id which
+            # GL's API uses instead of slugs like GH
+            self.get_client().check_file(project_id, filepath, version)
+        except ApiError as e:
+            if e.code != 404:
+                raise
+            return None
+
+        base_url = self.model.metadata["base_url"]
```

Comment:
@markstory I'm getting a little confused with GitLab, do you happen to know if this is correct thing to use to generate the url down below?

Basically we wanna send them to something like: 

`"https://gitlab.com/gitlab-org/gitlab/blob/master/README.md"`
 
plus a line no. which we can append in the frontend